🐛 BUG: Cannot find package "@babel/types" on the CLI

### What version of `astro` are you using?

v1.0.0-beta.64

### Are you using an SSR adapter? If so, which one?

None

### What package manager are you using?

yarn

### What operating system are you using?

Windows

### Describe the Bug

When creating a new project via `yarn astro create`, the following error is logged on any `astro` command:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '...\node_modules\@babel\types\' imported from ...\node_modules\astro\dist\core\add\babel.js
    at new NodeError (node:internal/errors:372:5)
    at legacyMainResolve (node:internal/modules/esm/resolve:317:9)
    at packageResolve (node:internal/modules/esm/resolve:895:14)
    at moduleResolve (node:internal/modules/esm/resolve:957:20)
    at defaultResolve (node:internal/modules/esm/resolve:1173:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:604:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:318:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

This may very well be an issue with my computer, as any other instances work, however other pre-existing Astro projects work on my computer.

### Link to Minimal Reproducible Example

N/A

### Participation

- [ ] I am willing to submit a pull request for this issue.

> I do remember seeing another package that may have been causing a similar issue, let me check really quick.

Found the package: `@babel/helper-validator-identifier`, I get a similar error:

```
Error: Cannot find module 'D:\Code\babeltestthing\node_modules\@babel\helper-validator-identifier\lib\index.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:954:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:947:15)
    at resolveExports (node:internal/modules/cjs/loader:482:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (D:\Code\babeltestthing\node_modules\@babel\types\lib\validators\isValidIdentifier.js:8:34)
    at Module._compile (node:internal/modules/cjs/loader:1099:14) {
  code: 'MODULE_NOT_FOUND',
  path: 'D:\\Code\\babeltestthing\\node_modules\\@babel\\helper-validator-identifier\\package.json'
}
```

--- a/packages/astro/package.json
+++ b/packages/astro/package.json
@@ -87,6 +87,7 @@
     "@astrojs/telemetry": "^0.2.4",
     "@astrojs/webapi": "^0.12.0",
     "@babel/core": "^7.18.2",
+    "@babel/types": "^7.18.4",
     "@babel/generator": "^7.18.2",
     "@babel/parser": "^7.18.4",
     "@babel/plugin-transform-react-jsx": "^7.17.12",
@@ -137,7 +138,6 @@
     "zod": "^3.17.3"
   },
   "devDependencies": {
-    "@babel/types": "^7.18.4",
     "@playwright/test": "^1.22.2",
     "@types/babel__core": "^7.1.19",
     "@types/babel__generator": "^7.6.4",

